lib/data_update_scripts/20201217184442_append_community_to_community_name.rb

```diff
@@ -0,0 +1,9 @@
+module DataUpdateScripts
+  class AppendCommunityToCommunityName
+    def run
+      SiteConfig.where(var: "community_name").find_each do |community_name|
+        community_name.update!(value: "#{community_name.value} Community")
```

Comment:
Thinking about this some more and re-reading the code...

Should we actually be updating the `community_name` to take `collective_noun_disabled` into account?

We used to rely on and render `community_qualified_name` everywhere, which did this:
https://github.com/forem/forem/blob/978b698e7888386f5acf8e907a0ea6b6ffadbb46/app/helpers/application_helper.rb#L176-L180

If the `collective_noun_disabled` was _not_ disabled (aka if it was `false`) we used to build up the community name by interpolating `community_name` and the `SiteConfig.collective_noun`. But, if `collective_noun_disabled` _was_ `true`, then we just used to return `community_name`.

If we just directly use what we have in this data update script:
```ruby
community_name.update!(value: "#{community_name.value} Community")
```

then we are no longer taking that `collective_noun_disabled` into account, which means that any Forem instance that had _not_ disabled the collective noun feature would have `"Community"` appended to it...and that might be an unexpected change that would take those Forem creators by surprise! 😅 

I think could be clever with this data update script and _use_ the `collective_noun_disabled` concept to figure out if we even _need_ to append `"Community"` to the end of the `community_name`. Of course, we'd have to do this _before_ we destroy those fields from the SiteConfig, so order of operations would be essential here.

If we want to reduce the friction and any unexpected community name changes for _all_ Forem instances, I _think_ we want to do this in the data update script:
1) Check the value of `collective_noun_disabled` for the Forem's SiteConfig
2) If the "collective noun" concept has been disabled (`collective_noun_disabled` is `true`)
`collective_noun_disabled`, then we don't need to update anything
3) But if the "collective noun" concept has _not_ been disabled (`collective_noun_disabled` is `false`), then we want to update the `community_name` to be: `"#{community_name} #{SiteConfig.collective_noun}`, which matches [what we do](https://github.com/forem/forem/blob/master/app/helpers/application_helper.rb#L177) in the `community_qualified_name` method (the method that you are eliminating in this PR!)

The other reason I bring this up is that, currently, this data update script is appending the hard-coded string, `"Community"`. But we have this concept of `collective_noun` in the `SiteConfig`, and if there is any Forem out there that overrode the default value of `collective_noun`, we won't be taking that into account with this hardcoded string. 🤔

Reply:
This all makes sense to me! Thank you for the detailed explanation. 😄  I'll make these changes to the `data_update` script and remove the `data_update` script that removes `collective_noun` and `collective_noun_disabled` into a separate PR to ensure proper removal and re-request your review @vaidehijoshi!

Reply:
I've made this change! The newly refined `data_update` script that checks whether `collective_noun_disabled` is true or false and updates the `community_name` based on that value has been updated in this PR, while the `data_update` script that removes `collective_noun` and `collective_noun_disabled` from the SiteConfig has been removed from this PR and opened as a separate PR, which can be found [here](https://github.com/forem/forem/pull/11973).

cc @vaidehijoshi @benhalpern